src/lib/components/Search/index.js

```diff
@@ -306,14 +305,23 @@ class Search extends BaseStateElement {
     this.updateComplete.then(() => {
       this.inputEl.focus();
     });
+
+    // See https://github.com/GoogleChrome/web.dev/issues/8131
+    /** @type {HTMLElement|object} */
+    const navDrawer = document.querySelector('web-navigation-drawer') || {};
+    navDrawer.inert = true;
```

Comment:
I think the drawer should react to isSearchExpanded: true change of state instead. Here's the state listener for drawer: https://github.com/GoogleChrome/web.dev/blob/f5ccf1921c92cc60fc21146441b6b733863d8d3e/src/lib/components/NavigationDrawer/index.js#L89

Since search already updates that field in the state store, Drawer can use it (instead of having inert state set from outside.)